EssentialCSharp.Web/Views/Shared/_Layout.cshtml

```diff
@@ -302,25 +277,25 @@
     @* Recursive vue component template for rendering the table of contents. *@
     <template id="toc-tree">
         <li v-if="item.items.length">
-            <details :open="expandedTocs.has(item.key)"
+            <details :open="expandedTocs.has(item.keys)"
                      v-on:toggle="!$event.target.open ? expandedTocs.delete(item.key) : expandedTocs.add(item.key)">
                 <summary :class="{
                          'toc-content' : item.level==0,
                          'nested' : item.level>
                     0,
-                    'current-section': currentPage.some(p => p.key == item.key),
+                    'current-section': currentPage.some(p => JSON.stringify(p.keys) == JSON.stringify(item.keys)),
```

Comment:
It seems that in other places we're using the first key of a page as something of a canonical key for a page. Can we not just use the first key here as well rather than repeatedly stringifying arrays?

Reply:
That definitely makes sense. My reasoning behind trying to find another implementation is I didn't want to assume Keys was non-empty. My only other idea is to add a helper function to check the arrays length then return first element otherwise null, but I wasn't sure if that was the best implementation. Is that what you're thinking for a better alternative?

Also, it's super cool to get feedback from the main maintainer of Coalesce. It's one of my favorite technologies/ideas, maybe aside from LINQ. So yeah, it's cool to virtually meet you :)